spec/hosts/nodes/tang01.dev.lsst.org_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'tang01.dev.lsst.org', :sitepp do
+  on_supported_os.each do |os, facts|
+    next unless os =~ %r{almalinux-9-x86_64}
+
+    context "on #{os}" do
+      let(:facts) do
+        override_facts(facts,
+                       fqdn: 'tang01.dev.lsst.org',
+                       is_virtual: true,
+                       virtual: 'kvm',
+                       dmi: {
+                         'product' => {
+                           'name' => 'KVM',
+                         },
+                       })
+      end
+      let(:node_params) do
+        {
+          role: 'tang',
+          site: 'dev',
+        }
+      end
+
+      it { is_expected.to compile.with_all_deps }
+
+      include_examples 'vm'
+      include_context 'with nm interface'
```

Comment:
Nice, that's rarely used/known.

Reply:
I need to write a rspec extension gem for testing that puppet mod... some day I will find the time...